test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -523,6 +526,156 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, Label(string(lab
 					Expect(ovnContainerCpusetAfterReboot).To(Equal(cpumask), "affinity of ovn kube node pods(%s) do not match with ovservices(%s)", ovnContainerCpusetAfterReboot.String(), cpumask.String())
 				}
 			})
+
+			// Automates OCPBUGS-35347: ovs-vswitchd is using isolated cpu pool instead of reserved pool
+			It("[test_id:75257] verify ovs-switchd threads inherit cpu affinity", Label("ovs1"), func() {
+				checkCpuCount(context.TODO(), workerRTNode)
+				By("Get Thread Affinity of ovs-vswitchd process")
+				//Get ovs-switchd thread affinity
+				threadAffinity, err := ovsSwitchdThreadAffinity(ctx, workerRTNode)
+				Expect(err).ToNot(HaveOccurred())
+
+				// Verify thread affinity contains all the online cpu's
+				for _, line := range threadAffinity {
+					if line != "" {
+						cpumask := strings.Split(line, ":")
+						threadsCpuset, err := cpuset.Parse(strings.TrimSpace(cpumask[1]))
+						Expect(err).ToNot(HaveOccurred())
+						Expect(threadsCpuset).To(Equal(onlineCPUSet))
+					}
+				}
+
+				// create deployment with 2 replicas and each pod have 2 cpus
+				var dp *appsv1.Deployment
+				// create a deployment to deploy gu pods
+				dp = newDeployment()
+				testNode := make(map[string]string)
+				testNode["kubernetes.io/hostname"] = workerRTNode.Name
+				dp.Spec.Template.Spec.NodeSelector = testNode
+				err = testclient.DataPlaneClient.Create(ctx, dp)
+				Expect(err).ToNot(HaveOccurred(), "Unable to create Deployment")
+
+				// Delete deployment
+				defer func() {
+					// delete deployment
+					By("Delete Deployment")
+					testlog.Infof("Deleting Deployment %v", dp.Name)
+					err := testclient.Client.Delete(ctx, dp)
+					Expect(err).ToNot(HaveOccurred())
+				}()
+
+				testlog.Info("Get the pods list form the deployment")
+				// Get pods from the deployment
+				listOptions := &client.ListOptions{
+					Namespace:     dp.Namespace,
+					FieldSelector: fields.SelectorFromSet(fields.Set{"spec.nodeName": workerRTNode.Name}),
+					LabelSelector: labels.SelectorFromSet(labels.Set{"type": "telco"}),
+				}
+				podList := &v1.PodList{}
+				Eventually(func() bool {
+					if err := testclient.DataPlaneClient.List(context.TODO(), podList, listOptions); err != nil {
+						return false
+					}
+					if len(podList.Items) < 1 {
```

Comment:
nit: `newDeployment()` has Replicas: 2 . Is it ok to wait for only one pod to be ready?
Maybe we can look at the number of ReadyReplicas on the deployment, like L627. WDYT?

Reply:
Addressed in the latest commit